classes/Guards/CapGuard.php

```
@@ -0,0 +1,53 @@
+	}
+
+	protected static function secretKey(): string|null
+	{
+		return DreamForm::option('guards.cap.secretKey');
+	}
+
+	public function run(): void
+	{
+		$data = [
+			'secret' => static::secretKey(),
+			'response' => SubmissionPage::valueFromBody('cap-token')
```

PHPMD (2.15.0)

Avoid using static access to class '\tobimori\DreamForm\Models\SubmissionPage' in method 'run'. (Clean Code Rules)

(StaticAccess)